.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,45 +1,56 @@
 ---
 name: Bug report
-about: Create a report with a reproducible test case in a JSFIDDLE, For anything else use the github DISCUSSIONS feature. Anything different than a bug report will be closed automatically.
+about: Create a detailed bug report with a reproducible test case. Anything else will be closed, consider using [DISCUSSIONS](https://github.com/fabricjs/fabric.js/discussions). 
+
 ---
 
-<!-- ISSUES THAT ARE NOT BUGS OR LACK A TEST CASE WILL BE CLOSED. -->
+## Check List
 
-<!-- BUG TEMPLATE -->
+- [ ] I have read and followed the [Contributing Guide](/CONTRIBUTING.md)
+- [ ] I have read and followed the [Issue Tracker Guide](/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
+- [ ] I have searched and referenced existing issues and discussions
+- [ ] I have managed to reproduce the bug **after** upgrading to the **latest** version
```

Comment:
or latest version of the same major version ?

Reply:
Why actually?
Do we want that?
Supporting old majors?
I don't